src/Storage/Query/Handler/SelectQueryHandler.php

```diff
@@ -15,7 +16,7 @@ class SelectQueryHandler
     /**
      * @param ContentQueryParser $contentQuery
      *
-     * @return QueryResultset
+     * @return QueryResultset|Content|null
```

Comment:
1) dump output for certain queries
2) its used for queries so should return set or content or null
3) look at line 51 & 53